demo/meeting/src/app.tsx

```diff
@@ -62,4 +64,27 @@ const Theme: React.FC = ({ children }) => {
   );
 };
 
+function useQuery() {
+  return new URLSearchParams(useLocation().search);
+}
+
+const DemoMeetingProvider: React.FC = ({ children }) => {
+  const query = useQuery();
+  const querylogLevel = query.get('logLevel');
```

Comment:
I think we could just keep the original <MeetingProvider> in the `App` above and calculate this logLevel value outside of the component, since it's only occurring when we initialize MeetingManager when we first load the app

```
const params = new URLSearchParams(window.location.search);
const logLevel = urlParams.get('logLevel') || 'warn';

const App = () => {
 <MeetingProvider logLevel={logLevel}>

 </MeetingProvider>
}
```

Reply:
Updated to:
```
const urlParams = new URLSearchParams(window.location.search);
const queryLogLevel = urlParams.get('logLevel') || 'warn';
const logLevel = SDK_LOG_LEVELS[queryLogLevel] || SDK_LOG_LEVELS['warn'];
```